test/servinge2e/servicemesh_test.go

```diff
@@ -336,6 +337,16 @@ func jwtHttpGetRequestBytes(url string, token *string) (*http.Response, []byte,
 	return resp, body, err
 }
 
+// randomSuffix generates a random suffix suitable for making unique k8s resource names
+func randomSuffix(t *testing.T) string {
```

Comment:
We could probably reuse https://github.com/knative/pkg/blob/master/test/helpers/name.go#L59

Reply:
OK, I'll wait for https://github.com/openshift-knative/serverless-operator/pull/432 first then, as that would conflict with the Gopkg and vendoring changes there.